pkgs/development/libraries/gegl/4.0.nix

```diff
@@ -58,11 +58,8 @@ stdenv.mkDerivation rec {
       sha256 = "1psls61wsrdq5pzpvj22mrm46lpzrw3wkx6li7dv6fyb65wz2n4d";
     })
 
-    # Fix test timeout. Downstream debian patch.
-    (fetchpatch {
-      url = "https://salsa.debian.org/gnome-team/gegl/raw/9b7520b38d87cd8ad4b39bf0b8c62d011da25169/debian/patches/increase_test_timeout.patch";
-      sha256 = "1prc1h1aipjd9db0i1j7nzga4zvk3vl8qsjpz1jzv1wwvz02isly";
-    })
```

Comment:
I think this patch is fixing a different, slower test? ie: we might want to keep both

Reply:
I suppose so, reverting that part.

Reply:
Done :)